sections/03_categorize.md

```diff
@@ -540,9 +541,9 @@ parameter which provides a quantification of privacy. Simmons et al.
 [@doi:10.1016/j.cels.2016.04.013] present the ability to perform GWASs in a
 differentially private manner and Abadi et al. [@arxiv:1607.00133] show the
 ability to train deep learning classifiers under the differential privacy
-framework. Federated learning
-[@url:http://proceedings.mlr.press/v54/mcmahan17a.html] and secure aggregations
-[@url:https://eprint.iacr.org/2017/281.pdf @tag:Papernot2017_pate] are
+framework. Federated learning [@url:http://proceedings.mlr.press/v54/mcmahan17a.html] and secure aggregations
+[@url:http://proceedings.mlr.press/v54/mcmahan17a.html
```

Comment:
Is `url:http://proceedings.mlr.press/v54/mcmahan17a.html` an example of both federated learning and secure aggregations?  I quick skim suggested that secure aggregations build on top of this paper, but I admittedly didn't read it.